app/Helpers/InstanceHelper.php

```diff
@@ -60,4 +62,14 @@ public static function getChangelogEntries($limit = null)
 
         return $changelogs;
     }
+
+    /**
+     * Check if the instance has at least one account.
+     *
+     * @return bool
+     */
+    public static function hasAtLeastOneAccount(): bool
+    {
+        return DB::table('accounts')->count() > 0;
+    }
```

Comment:
We could create this function as `hasOnlyOneAccount`, as almost all call to this function are like `! hasAtLeastOneAccount()`

Reply:
I completely agree BUT it’s one more change in an already complex PR and I would prefer make this change in a separate PR.
I still need to fine tune the Account model to remove everything about the population of data anyway.